docs/src/layouts/MainLayout.vue

```diff
@@ -1,74 +1,80 @@
 <template>
   <q-layout view="hHh lpR fff" class="bg-lp-dark">
-
-    <q-header class="bg-lp-dark">
-      <q-toolbar>
+    <q-header class="bg-lp-dark horizontal-glow-bar">
```

Comment:
This header should show also on docs, as per mockup, to keep the UI consistent
But I see now Nick haven't re-mapped some of the current links in the new header, so let's keep this for a followup PR